README.md

```diff
@@ -34,6 +34,16 @@ use ClaudioDekker\WordGenerator\Generator;
 echo Generator::generate('-'); // Outputs 'autumn-firefly', 'crimson-meadow', etc.
 ```
 
+If you need words of a certain length, the second argument can be used to set how long the word will become. It does this by adding more adjectives.
+It's also possible to use a custom separator character, by passing it as the first argument:
+```php
+<?php
+
+use ClaudioDekker\WordGenerator\Generator;
+
+echo Generator::generate('-', 5); // Outputs 'autumn-firefly' etc.
```

Comment:
According to this, using `5` as the length still just generates "autumn-firefly", which seems wrong?